requirements/test.txt

```diff
@@ -2,6 +2,7 @@ pytest
 pytest-asyncio
 pytest-cov
 pytest-deadfixtures
+pytest-runner
```

Comment:
Can you give me more context about this?

Reply:
It's required on `setup.py` at line 91 
```
python setup_requires=['pytest-runner'],
```
The IDE show an warnning `Package requirement 'pytest-runner' is not satisfied`.

Reply:
I think it's your IDE, I could not find anything in regular environments or circleci. Please revert this change.